lib/fluent/plugin/in_tail.rb

```diff
@@ -59,6 +59,7 @@ def initialize
       @shutdown_start_time = nil
       @metrics = nil
       @startup = true
+      @capability = Fluent::Capability.new(:current_process)
```

Comment:
memo: When capng gem isn't installed, stub FLuent::Capability is defined:
https://github.com/fluent/fluentd/blob/0337ea9dcc7e6871826f1c5063de34be4a04ba74/lib/fluent/capability.rb#L58